<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPage</class>
 <widget class="QWidget" name="TestPage">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>752</height>
    </rect>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QStackedWidget" name="questionSlideShow">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>831</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #fff;
	border-radius: 12px;
}</string>
     </property>
    </widget>
    <widget class="QFrame" name="questionChooser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>181</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #fff;
	border-radius: 15px;
	border: 0px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>There're [] questions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>680</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
	background-color: rgb(45, 18, 153);
   border-radius: 12px;  
   color: #fff;
   font-family: Inter,sans-serif;
   font-size: 1.2rem;
   font-weight: 700;
   line-height: 1;
   margin: 0;
   outline: none;
   padding: 1rem 1.2rem;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
   border: 0;
 }

QPushButton:hover:!pressed
{
	background-color: rgb(75, 45, 170);
	border: 1.5px solid rgb(75, 45, 170);
}

QPushButton:hover {
	border: 1.5px;	
}</string>
     </property>
     <property name="text">
      <string>Submit!</string>
     </property>
    </widget>
    <widget class="QFrame" name="rightSideBar">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>30</y>
       <width>221</width>
       <height>621</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: rgba(240, 240, 240, 255);
	border-radius: 15px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>210</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="avtLbl">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="role">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>236</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="courseName">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>300</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>course name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="test">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>330</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>test name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="timer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	background-color: rgb(45, 18, 153);
   	border-radius: 12px;  
	color: #fff;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="questionTag">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>821</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Question 1: intro</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>rightSideBar</zorder>
    <zorder>questionChooser</zorder>
    <zorder>questionSlideShow</zorder>
    <zorder>questionTag</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
